.env

```diff
@@ -0,0 +1 @@
+HTTPPORT=80
```

Comment:
why put this in `.env` rather than wherever the other environment variable, `APP_URL` is coming from?